src/mjolnir/ingest_transit.cc

```diff
@@ -90,14 +90,18 @@ struct tile_transit_info_t {
 
 struct feed_cache_t {
   std::unordered_map<std::string, gtfs::Feed> cache;
+  std::string gtfs_dir;
+
+  feed_cache_t(const std::string& gtfs_dir) : gtfs_dir(gtfs_dir) {
+  }
```

Comment:
so this is the first sign for a bigger change: when parsing all feeds and collect their info per tile, we added uniqueness of the stop/trip etc objects by adding a feed's _absolute_ directory path. that's crazy wasteful, so I changed it to be only the directory name, i.e. instead of `{stop_id, "/home/nilsnolde/dev/cpp/valhalla/site/transit_feeds/Jena"`}`, it's now `{stop_id, "Jena"}`.

so here, we have to get the parent directory into the cache constructor, so we can still load the feeds properly.